### ISSUE TITLE
Extension not working on Chrome

### ISSUE THREAD

**Issue by explorer326** [james-fray/tab-reloader#120]
The countdown timer counts down to zero and just resets; nothing happens to the tab. This has been happening since the latest Chrome plugin update where the interface changed. Tried to reapply allowing host permissions but this had no effect. Chrome is on Version 101.0.4951.54

**Comment by james-fray** [comment]
Is this happening for all websites? Can you confirm on a clean profile?
https://support.google.com/chrome/answer/2364824?hl=en&co=GENIE.Platform%3DDesktop#:~:text=Add%20a%20new%20profile,name%2C%20photo%20and%20color%20scheme.

**Comment by 407pilot** [comment]
I can confirm this is an issue. Removing, then reinstalling, tab reloader fixed the issue for me.

**Comment by explorer326** [comment]
> Is this happening for all websites? Can you confirm on a clean profile? https://support.google.com/chrome/answer/2364824?hl=en&co=GENIE.Platform%3DDesktop#:~:text=Add%20a%20new%20profile,name%2C%20photo%20and%20color%20scheme.

Yes, it is happening for all websites on a fresh reinstall, new profile. Removing and reinstalling tab reloader did not fix the issue for me.

**Comment by james-fray** [comment]
@explorer326, Since I am not able to reproduce this issue, let's debug:

Open chrome://extensions, and turn on the developer mode. Click on the "inspect" button for this extension and switch to the console tab. Now try to set a job and see if you get an error in the console.

**Comment by JustaNormalGuy230** [comment]
FWIW:

I've experienced this behavior on a particular (older) CGI-Coded Web Site using ANOTHER tab re-loader (refresher) extension. (I cannot say much about whether it happens with this particular tab reloading extension, but with all the reloading options James provides, I have a hunch that one of them will correct this type of issue.).  When I get a few moments of free  time, I'll switch over a to a fresh/test browser installation and check it out, getting familiar with this particular tab re-loader.

... Is this even related to your particular issue, explorer326?  Not sure/hopeful, but we'll get another data point to possibly sleuth it out.  I believe in my case, I have not even tried to re-install this other extension I'm using and that I may have YET  ANOTHER refreshing extension installed (that I sometimes run tests on), but [hopefully] de-activated (yes, I do a lot of crazy testing).

**Comment by 407pilot** [comment]
Following up my earlier comment, the extension has stopped working again. 

After enabling developer mode I do not see any errors in the console.  However, I don't see _anything_ in the console even when I select verbose. Not sure if I did something wrong. 

When I look at the extension it is performing a countdown, and adds the page to the number of pages being reloaded, but the icon does not turn orange as it does for other pages. I don't know if that means anything.

**Comment by james-fray** [comment]
> the extension has stopped working again.

It looks like the extension somehow missed the tabs with the active jobs. Have you done a restart? Please try to find a way to reproduce the bug and report back.

The thing that I don't understand is even if the extension cannot find a tab, on the next timeout, it tries to locate it once again, and if it fails pauses the job until a matching tab is available.

> but the icon does not turn orange as it does for other pages

You get the list of active tabs in the bottom section. What happens when you click on one?

**Comment by 407pilot** [comment]
>It looks like the extension somehow missed the tabs with the active jobs. Have you done a restart? Please try to find a way to reproduce the bug and report back.

It is very reproducible. It happens every time I open Chrome on one particular website. Is there a way to get debug data out? The website is behind a login, so I don't know any other way to get you information. 

>You get the list of active tabs in the bottom section. What happens when you click on one?

I was going to try, but for whatever reason it works without an issue on Chrome on my Macbook. The computer I'm having an issue with is Windows 10. I will update this post with additional information once I get back to work.

**Comment by james-fray** [comment]
> It is very reproducible.

Can you reproduce on another website? Please give the dev version a try and report back:

1. Open chrome://extensions in a browser tab and turn the developer mode on.
2. Download https://github.com/james-fray/tab-reloader/archive/refs/heads/master.zip
3. Drop "v3" directory to the chrome://extensions page

**Comment by Karkir72** [comment]
I tried this. There are no errors in the console. But V3 just doesn't refresh the page and becomes pink. I noticed that sometimes the page does update with the regular version of the extension. If you put a period of 10 seconds, it might refresh once every minute or so.
